CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# 📖 Contributing rules in `ufpy` repository
+
+If you want to contribute, you have to read this rules and follow they.
+
+## 💻 Code style
+
+- Formatter: `black`
+- Optimizer of imports: `isort`
+- Linter: `Pylint`
+- Generic style: `True`
+
+Generic style is style where are using `typing.Generic` class in all `useful classes`
+
+For example,
+```python
+from typing import Generic, TypeVar
+
+T = TypeVar('T')
+
+class A(Generic[T]): 
+    def __init__(self, a: T): ...
+```
+
+- `@overload`s: `True`
+
+You must define `@typing.overload`s in public methods when your method has two and more ways to define arguments
```

Comment:
**Use decorator** `@typing.overload`s in public methods **if** your method has two and more **variants** to define arguments